lib/hanami/environment.rb

```diff
@@ -48,11 +48,17 @@ class Environment
     # @api private
     RACK_ENV_DEPLOYMENT = 'deployment'.freeze
 
-    # Default `.env` per environment file name
+    # Default `.env` files that are loaded. The entries are ordered from highest
+    # to lowest priority.
     #
     # @since 0.2.0
     # @api private
-    DEFAULT_DOTENV_ENV = '.env.%s'.freeze
+    DEFAULT_DOTENV_ENV_PATTERNS = [
+      '.env.%{environment}.local'.freeze,
+      '.env.local'.freeze,
+      '.env.%{environment}'.freeze,
+      '.env'.freeze
```

Comment:
@gmanley Is `.env` here really needed?

Reply:
Are you suggesting not loading `.env` at all? Do you prefer to have less options and guide people to always use .env.local or environment specific ones?

Reply:
@gmanley Fine with me, thanks for the clarification. 👍